server/stream.go

```diff
@@ -4620,8 +4624,8 @@ func (mset *stream) numConsumers() int {
 // Lock should be held.
 func (mset *stream) setConsumer(o *consumer) {
 	mset.consumers[o.name] = o
-	if o.cfg.FilterSubject != _EMPTY_ {
-		mset.numFilter++
+	if o.subjf != nil {
+		mset.numFilter += len(o.subjf)
```

Comment:
This just is number of filtered consumers, so still could be just ++ here.

so would suggest.

```go
if len(o.subjf) > 0 {
  mset.numFilter++
}
```

Reply:
ah! Thought it's number of filters. 
Fixed and added comment to struct field.